spec/lib/thredded/content_formatter_spec.rb

```diff
@@ -108,6 +108,28 @@ def parsed_html(html)
       end
     end
 
+    context 'with no onebox caching' do
+      around do |example|
+        onebox_views_cache = Thredded::HtmlPipeline::OneboxFilter.onebox_views_cache
+        onebox_data_cache = Thredded::HtmlPipeline::OneboxFilter.onebox_data_cache
+        Thredded::HtmlPipeline::OneboxFilter.onebox_views_cache = ActiveSupport::Cache::MemoryStore.new
+        Thredded::HtmlPipeline::OneboxFilter.onebox_data_cache = ActiveSupport::Cache::MemoryStore.new
+        example.run
```

Comment:
I'm not sure whether the code after this line runs in case of a test failure. It might be necessary to use `ensure` here.

Reply:
Nope. it's part of the `around` guarantee to trap exceptions. https://relishapp.com/rspec/rspec-core/v/3-7/docs/hooks/around-hooks#an-around-hook-continues-to-run-even-if-the-example-throws-an-exception